cmd/tk/export.go

```diff
@@ -29,13 +29,17 @@ func exportCmd() *cli.Command {
 
 	vars := workflowFlags(cmd.Flags())
 	getExtCode := extCodeParser(cmd.Flags())
+	allowSubFolderCreation := cmd.Flags().Bool("allow-subfolder-creation", false, "Allow export to create subfolders in the out directory if the format contains '/'")
 	format := cmd.Flags().String("format", "{{.apiVersion}}.{{.kind}}-{{.metadata.name}}", "https://tanka.dev/exporting#filenames")
 	extension := cmd.Flags().String("extension", "yaml", "File extension")
 
 	templateFuncMap := template.FuncMap{
 		"lower": func(s string) string {
 			return strings.ToLower(s)
 		},
+		"replace": func(old, new string, s string) string {
+			return strings.Replace(s, old, new, -1)
+		},
```

Comment:
Note for future PR's: This is the second function added here that lines up with `masterminds/sprig`, so consider using that library in case we get more of those.

No need to do that for this one though, it's fine as is.